mmv1/products/cgc/terraform.yaml

```diff
@@ -48,6 +48,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "spot_vm_instance"
         vars:
           spot_instance_name: "spot-instance-name"
+    ### Custom Hostname VM ###
+      - !ruby/object:Provider::Terraform::Examples
+        name: "instance_custom_hostname"
+        primary_resource_name: "google_compute_instance"
```

Comment:
```suggestion
```

I don't think this example needs this set. It needs to match the resource `name` field exactly (well, not quite- it has to match the final rendered value, which is generally a little different), causing the current error if it doesn't:

```
provider_test.go:309: Resource specified by ResourceName couldn't be found: google_cgc_snippet.custom_hostname_instance
```

Reply:
Change "primary_resource_name" to "primary_resource_type" which is what it should have been in the first place. Will let tests run and check again. Thanks for the help